pkg/restore/client.go

```diff
@@ -305,6 +334,70 @@ func (rc *Client) CreateTables(
 	return rewriteRules, newTables, nil
 }
 
+// RemoveTiFlashReplica removes all the tiflash replicas of a table
+// TODO: remove this after tiflash supports restore
+func (rc *Client) RemoveTiFlashReplica(tables []*utils.Table, placementRules []placement.Rule) error {
+	schemas := make([]*backup.Schema, 0, len(tables))
+	for _, table := range tables {
+		if rule := utils.SearchPlacementRule(table.Info.ID, placementRules, placement.Learner); rule != nil {
+			table.TiFlashReplicas = rule.Count
+		}
+		tableData, err := json.Marshal(table.Info)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		dbData, err := json.Marshal(table.Db)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		schemas = append(schemas, &backup.Schema{
+			Db:              dbData,
+			Table:           tableData,
+			Crc64Xor:        table.Crc64Xor,
+			TotalKvs:        table.TotalKvs,
+			TotalBytes:      table.TotalBytes,
+			TiflashReplicas: uint32(table.TiFlashReplicas),
+		})
+	}
+
+	// Update backup meta
+	rc.backupMeta.Schemas = schemas
+	backupMetaData, err := proto.Marshal(rc.backupMeta)
```

Comment:
the origin backupmeta schemas has changed to new-created schemas, but it seems has no affect on next restore. will it affect on our current debug procedure?